docs/services/services.md

```diff
@@ -0,0 +1,35 @@
+# Overview
+
+Qlik Core consists of a core set of supported services, including the Qlik Associative Engine:
```

Comment:
I would remove: "including the Qlik Associative Engine:"
Don't think it's needed. Why is QAE included and not the other?

Reply:
Was copy/paste from Download page, but I agree, removing